js/tji-datasets-explore.js

```diff
@@ -57,10 +59,10 @@ TJIGroupByBarChart.prototype.create = function(data) {
   // of groupBy keys now, in their sorted order.
   this.ordered_keys = grouped.keys
 
-  var $canvas = jQuery('<canvas class="tji-chart-canvas" height="1" width="1"/>');
+  var $canvas = jQuery('<canvas class="tji-chart__canvas" height="1" width="1"/>');
```

Comment:
Could you enlighten me as to what naming convention you're using?

Reply:
[BEM](http://getbem.com/introduction/)